api/announcements.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express");
+const models = require("./models");
+const utils = require("./utils");
+const router = express.Router();
+const Sentry = require("@sentry/node");
+const Busboy = require("busboy");
+const AWS = require("aws-sdk");
+
+router.use(utils.authMiddleware);
+
+router.get("/", async (req, res) => {
```

Comment:
@tjram ideally this can take in the role the announcements are targeted at, so we don't leak them on the client by accident. If no role is provided we can return all